packages/framework/bundle/heatmap/bundle.js

```diff
@@ -29,7 +29,10 @@ Oskari.clazz.define("Oskari.mapframework.bundle.heatmap.HeatmapBundle", function
             "src": "../../../../bundles/framework/heatmap/HeatmapDialog.js"
```

Comment:
You moved the files from framework to mapping, you should update the folders in here too.